core/modules/block/block.admin.inc

```diff
@@ -137,6 +137,16 @@ function block_admin_configure($form, &$form_state, $delta = NULL) {
     '#type' => 'submit',
     '#value' => t('Save block'),
   );
+  if($delta)
+  {
```

Comment:
The formatting should match Backdrop's standards https://api.backdropcms.org/php-standards#control-structures : space after `if`, the open bracket should on be the same line, indent array items only two spaces, remove empty line in array, closing bracket on array should be indented the same as the `$form`.

Reply:
oops, sorry, I'll fix that

Reply:
I adjusted the white space.